cmd/servicelog/list.go

```diff
@@ -34,6 +34,10 @@ var listCmd = &cobra.Command{
 			}
 		}()
 
+		if len(args) != 1 {
+			log.Infof("The amount of arguments passed are: %d and they are '%v'", len(args), args)
```

Comment:
```suggestion
			log.Infof("Too many arguments. Expected 1m got %d", len(args))
```

Reply:
Ah this clarifies my question I had!

Reply:
@fahlmant  shouldn't it be:"Too many arguments. Expected 1, but got %d"? Is the `1m` a typo?

Reply:
yep, 1m is a typo. Fixed. And imo we should exit